src/Settings/genSettings.py

```diff
@@ -2,11 +2,15 @@
 # -*- coding: utf-8 -*-
 
 import sys
-import json
-
+import json # for json.dumps()
+import yaml
+import re
 
 def writeHelper(f, obj, pre, indent):
     ids = "    " * indent
+    cst = " const"
+    if indent == 1:
+        cst = ""
```

Comment:
Why does this file still exists if it is removed by `genSettings.cpp`?

Reply:
This file should be deleted correct?

Reply:
Yes. This file could be used to check the c++ genSettings, so I don't remove it yet.

Reply:
If it's fine you can remove it